aidants_connect_web/templates/footer/faq.html

```diff
@@ -0,0 +1,120 @@
+    <p>
+      Vous pouvez nous contacter sur
+      <a href="mailto:contact@aidantsconnect.beta.gouv.fr" target="_blank" rel="noopener">
+        contact@aidantsconnect.beta.gouv.fr
+      </a>
+    </p>
+
+    <h2 id="A-qui-s’adresse-AC-">A qui s’adresse Aidants Connect ?</h2>
+    <p>
+      Aidants Connect s’adresse des structures qui emploient des aidants professionnels  qui accompagnent régulièrement
+      des usagers dans la réalisation de leurs démarches en ligne.
+      Il peut s’agir de travailleurs sociaux, agents publics d’accueil, médiateurs numériques, etc.
+      Le service est aujourd’hui expérimenté par un nombre limité de structures.
+    </p>
+
+    <h2 id="aidants-habilités">Qui sont les aidants habilités à utiliser Aidants Connect ?</h2>
+    <p>
+      Les aidants habilités sont des aidants salariés d’une structure habilitée Aidants Connect.
+      Ils accompagnent des usagers dans la réalisation de leurs démarches administratives en ligne.</p>
+
+    <h2 id="AC-ouvert-a-tous-les-aidants-">Quand est-ce que Aidants Connect sera disponible pour tous les aidants ?</h2>
+    <p>Aidants Connect est pour le moment en phase de test avec 13 structures.
+      Si le service démontre un impact positif sur la sécurisation de la relation entre aidant et usager, il pourra
+      être ouvert à un plus grand nombre de structures et à une autre typologie d’aidants.</p>
+
+    <h2 id="structures-testeuses-">Quelles sont les structures expérimentatrices ?</h2>
+    <p>Voici la liste des structures expérimentatrices :</p>
+    <ul>
+      <li>Faitout Connecté&nbsp;(Communauté de communes Champagne-Picarde – Hauts-de-France)&nbsp;- structure labellisée France Services ;</li>
+      <li>L’Association des Centres Sociaux de Douai (Hauts-de-France)&nbsp;;</li>
+      <li>L’association de médiation numérique Net Solidaire (Charente-Maritime)&nbsp;;</li>
+      <li>La mairie de Préguillac (Charente-Maritime)&nbsp;;</li>
+      <li>La mairie de Rioux (Charente-Maritime)&nbsp;;</li>
+      <li>La Fabrique citoyenne de Floirac (Gironde)&nbsp;;</li>
+      <li>La Maison des Solidarités de Castanet-Tolosan (Communauté d’agglomération du SICOVAL – Haute-Garonne)&nbsp;;</li>
+      <li>La médiathèque Simone de Beauvoir (Valence Romans Agglomération – Drôme)&nbsp;;</li>
+      <li>Le tiers-lieu «&nbsp;La Palette&nbsp;» (Creuse) - structure labellisée France Services &nbsp;;</li>
+      <li>Le dispositif mobile «&nbsp;Fourgon Connecté&nbsp;» (Pyrénées-Atlantiques)&nbsp;;</li>
+      <li>Le CCAS d’Anglet (Pyrénées-Atlantiques)&nbsp;;</li>
+      <li>Le PIMMS de Blois (Centre-Val-de-Loire)-  structure labellisée France Services &nbsp;;</li>
+      <li>Un service d’action sociale du département d’Eure-et-Loir (Centre-Val-de-Loire).</li>
+      </ul>
+
+    <h2 id="combien-coûte-AC">Combien coûte Aidants Connect ?</h2>
+    <p>
+      Aidants Connect est gratuit pour les structures habilitées.
```

Comment:
Préciser qu'il s'agit d'un service public pour éclairer la réponse que l'on apporte à cette question ?
```suggestion
      Aidants Connect est un service public numérique dont l'utilisation est donc gratuite pour les structures habilitées.
```

Reply:
```suggestion
      Aidants Connect est un service public numérique gratuit pour les structures habilitées.
```
Sachant qu'il existe des services public numériques payants ?